ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-server-groups.j2

```diff
@@ -16,5 +15,6 @@ aaa group server {{ aaa_server_group.type }} {{ aaa_server_group.name }}
 {%                 endfor %}
 {%             endif %}
 {%         endif %}
+!
```

Comment:
I would move this `!` to before each server group. Then move the `!` in `aaa.j2` down before the three templates it tests for. This will be easier to maintain when we add something else.